### ISSUE TITLE
Eagle library

### ISSUE THREAD

**Issue by beckmx** [LilyGO/ESP32-MINI-D1-V2.0#10]
Hello guys, do you happen to have an eagle library?

**Comment by marcwolf1960** [comment]
Second that.  An Eagle library would be most welcome. I have several projects that I use this board in and I would love to make some PCB's for it to plug in to.

**Comment by sej7278** [comment]
yup, or kicad, i was promised DXF via twitter but not heard anything back
